packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -49,7 +49,7 @@ import {
 } from './visit-notes.resource';
 import styles from './visit-notes-form.scss';
 
-const allowedImageTypes = ['jpeg', 'jpg', 'png', 'webp'];
+const allowedImageTypes = ['.jpeg', '.jpg', '.png', '.webp'];
```

Comment:
This shouldn't be hard-coded. I think we can adopt the approach from the attachments ESM to load the allowed extensions from the global properties.

Reply:
@ibacher @NethmiRodrigo's handled that in https://openmrs.atlassian.net/browse/O3-3153. Presently, it's set to return the following `allowedFileExtensions` ['jpeg', 'jpg', 'png' , 'pdf']`. 

Any notable omissions?

Reply:
We've had to omit out 'webp' from the list of allowed file extensions since it wasn't a supported form of image as per `openmrs-module-attachments` (list of supported mime types were found from [here](https://github.com/openmrs/openmrs-module-attachments/blob/aca414441fbf0aa47fd7e991b190ed81d0d87581/api/src/main/java/org/openmrs/module/attachments/AttachmentsContext.java#L660))